<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="toolsFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="drawToolButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Draw</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/draw.png</normaloff>:/images/images/draw.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="selectToolButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/select.png</normaloff>:/images/images/select.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="moveToolButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/move.png</normaloff>:/images/images/move.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Tools</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="eraseToolButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Erase</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/effacer.png</normaloff>:/images/images/effacer.png</iconset>
     </property>
    </widget>
    <widget class="QWidget" name="drawOptions" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>120</width>
       <height>291</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>58</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Form</string>
      </property>
     </widget>
     <widget class="QComboBox" name="formComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Line</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectangle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ellipse</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>58</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
     <widget class="QComboBox" name="colorComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>White</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gray</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>58</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="widthSpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="styleGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>101</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Style</string>
      </property>
      <widget class="QRadioButton" name="solidStyleRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>So&amp;lid</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="dashedStyleRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Dashed</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="dottedStyleRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Dotted</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="dashDotStyleRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>DashDot</string>
       </property>
      </widget>
     </widget>
     <zorder>label_4</zorder>
     <zorder>formComboBox</zorder>
     <zorder>label_2</zorder>
     <zorder>colorComboBox</zorder>
     <zorder>label_4</zorder>
     <zorder>formComboBox</zorder>
     <zorder>label_2</zorder>
     <zorder>colorComboBox</zorder>
     <zorder>label_3</zorder>
     <zorder>widthSpinBox</zorder>
     <zorder>styleGroupBox</zorder>
    </widget>
   </widget>
   <widget class="QFrame" name="drawFrame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="zone_de_dessin" name="zoneDessin" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>751</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>zone_de_dessin</class>
   <extends>QWidget</extends>
   <header>zone_de_dessin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
